analyzers/src/SonarAnalyzer.CFG/ShimLayer/LightupHelpers.cs

```diff
@@ -39,6 +40,7 @@ public static class LightupHelpers
 
     public static bool SupportsIOperation => SupportsCSharp73;
 
+    [PerformanceSensitive("https://github.com/SonarSource/sonar-dotnet/issues/8106", AllowCaptures = false, AllowGenericEnumeration = false, AllowImplicitBoxing = false)]
```

Comment:
[education] why didn't you add this attribute on https://github.com/DotNetAnalyzers/StyleCopAnalyzers/pull/3711?

Reply:
The attribute/analyzer is not present in that repository.